specification/cognitiveservices/AnomalyDetector/main.tsp

```diff
@@ -49,6 +49,8 @@ namespace AnomalyDetector;
 @doc("The secret key for your Azure Cognitive Services subscription.")
 model AnomalyDetectorApiKeyAuth is ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">;
 
+@doc("The versions of this service")
```

Comment:
This should just be a repeat of the documentation-required suppression, as in each of the cases below.  There is no value added with these doc comments (I did this one first before I saw how many fo these there were, and that I would not have sensible documentation comments for all of them)

Reply:
This is a nit, though, the doc comments here also resolve the issue

Reply:
Fixed in https://github.com/Azure/azure-rest-api-specs/pull/24297